Skype/Trusted-Application-API/samples/QuickStartSamples/MessagingAfterJoinMeeting/App.config

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <startup> 
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+  </startup>
+  <appSettings>
+    <add key="AAD_ClientId" value="418fbf0c-d180-4cd9-8d13-7d0e9babbb9e" />
```

Comment:
We don't want to publish values, please replace these with stubs/placeholders like we have for other projects.